test/resources/integration.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet  type="text/xsl" href="odmlDocument.xsl"?>
+<odML version="1.1">
+  <id>e178c43c-8430-4a3e-bc0d-12e2b844b218</id>
+  <section>
+    <id>2fe02204-29eb-4358-884a-48c91755539e</id>
+    <name>sec_test_1</name>
+  </section>
+  <section>
+    <id>39ec5df0-f01d-4385-8865-1550a6b59c36</id>
+    <type></type>
+    <name>sec_test_2</name>
+  </section>
+  <section>
+    <id>27ca5a42-fc00-4342-af67-45d014c92d90</id>
+    <type>test</type>
+    <name>sec1</name>
+    <property>
+      <id>b237006e-a584-469d-adbc-e30dae9c4678</id>
+      <name>members</name>
+      <value>[-13,101,-11,0,-8]</value>
+      <type>string</type>
```

Comment:
Please refactor the test file; same for `json` and `yaml`:
a) please use a different naming scheme to make it clear to which test these files belong e.g. `validation_dtypes` and `validation_format`.
b) please move the content for missing section type to their own files and create distinct tests to make them independent.
c) please add as discussed two main sections with subsections. One with `type=string` everywhere and one with `type` missing completely. Add additional assertions accordingly to the same tests. It seems redundant, but I really would like to have both individual cases covered.